src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -260,9 +261,9 @@ private function compileBinaryOperator(Node\Expr\BinaryOp $node, CompilerContext
      * @param string          $constantName
      * @param ReflectionClass $class
      *
-     * @return CompilerContext|null
+     * @return ReflectionClass|null
```

Comment:
Only reason I'd leave some of these in is where the method has an actual description (as IDEs can complain about missing doc info for params/return types even tho they're there). I'd rather keep them in until IDEs be a bit more clever about it :) but yes, this one I've removed.